modules/nf-core/fqtk/main.nf

```diff
@@ -0,0 +1,51 @@
+process FQTK {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::fqtk=0.2.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/fqtk:0.2.0--h9f5acd7_0' :
+        'quay.io/biocontainers/fqtk:0.2.0--h9f5acd7_0' }"
+
+    input:
+    tuple val(meta), path(sample_sheet), val(fastq_readstructure_pairs)
+    // fastq_readstructure_pairs example:
+    // [[<fastq name: string>, <read structure: string>, <path to fastqs: path>], [example_R1.fastq.gz, 150T, ./work/98/30bc..78y/fastqs/]]
+
+    output:
+    // Demultiplexed file name changes depending on the arg '--output-types'
+    tuple val(meta), path('output/*.fq.gz')                         , emit: sample_fastq
+    tuple val(meta), path('output/demux-metrics.txt')               , emit: metrics
+    tuple val(meta), path('output/unmatched*.fq.gz')                , emit: most_frequent_unmatched
+    path "versions.yml"                                             , emit: versions
+
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def VERSION = '0.2.0'
```

Comment:
fqtk 0.2.1 is released, so we should be able to use the version arg now

Reply:
@nils, I will fix that now!